sbt-version-policy/src/main/scala/sbtversionpolicy/SbtVersionPolicySettings.scala

```diff
@@ -246,7 +246,31 @@ object SbtVersionPolicySettings {
             name.value,
           )
       }
-    }
+    },
+    versionPolicyVersionChange := {
```

Comment:
The fact that it is a key means that users can set its value. Is this expected? What if they set something inconsistent with the actual version change?

The thing that is still unclear to me is the developer workflow after a first major release: how do we tell the plugin that the next version change will be a major upgrade (e.g., if we work on a new major release), or a minor, or a patch upgrade?

Reply:
> The fact that it is a key means that users can set its value. Is this expected? What if they set something inconsistent with the actual version change?

If the user rewires a key and breaks it, then it's their responsibility, like any other keys in sbt. Having this here allows them/us to override the logic, and makes the plugin implementation clearer, I think.

Reply:
> The thing that is still unclear to me is the developer workflow after a first major release: how do we tell the plugin that the next version change will be a major upgrade (e.g., if we work on a new major release), or a minor, or a patch upgrade?

Suppose you're using sbt-dynver, then you can just put a tag like `2.0.0-M0` that you tag to denote the API breakage and not ship the tag. Another thing you can do is override `ThisBuild / version` as follows:

```scala
ThisBuild / dynverSonatypeSnapshots := true
ThisBuild / version := {
  val orig = (ThisBuild / version).value
  if (orig.endsWith("-SNAPSHOT")) "0.10.0-SNAPSHOT"
  else orig
}
```

Reply:
> Suppose you're using sbt-dynver, then you can just put a tag like `2.0.0-M0` that you tag to denote the API breakage and not ship the tag.

OK I see. It seems that we will have to change our habits: we can’t anymore rely on sbt-dynvar and push a tag to cut a release. We have to update the `version` just after we have cut a release, to prepare for the next one.